pipeline_dsl/concourse/task.py

```diff
@@ -109,8 +107,13 @@ def filter(tarinfo):
             return tarinfo
 
         for dir_concourse, dir_local in init_dirs:
-            dir_local = os.path.abspath(dir_local)
-            tar.add(dir_local, arcname=dir_concourse, filter=filter)
+            if isinstance(dir_local, list):
```

Comment:
`globed` is a list?

Reply:
Yes, because it can contain multiple files.